### PR TITLE
Added validation for the Qty, Length, width and height numeric fields

### PR DESCRIPTION
**Pull request by shivanibhanwal**
Length and pattern check is added to the Package template

### CODE REVIEW

File: app/templates/review_item/accept.hbs [hunk 1]
```diff
@@ -45,16 +45,16 @@
...
-                    {{input type="number" name="length" value=pkg.length placeholder=(t "placeholder.length") maxlength="8" }}
+                    {{numeric-input name="length" id=(concat 'length' index) value=pkg.length placeholder=(t "placeholder.length") maxlength="8" pattern="\d{1,8}"}}
                   </div>
                   <div class="small-2 columns ui inner">
-                    {{input type="number" name="width" value=pkg.width placeholder=(t "placeholder.width") maxlength="5" }}
+                    {{numeric-input id=(concat 'width' index) name="width" value=pkg.width placeholder=(t "placeholder.width") maxlength="5" pattern="\d{1,5}"}}
                   </div>
                   <div class="small-2 columns ui inner">
-                    {{input type="number" name="height" value=pkg.height placeholder=(t "placeholder.height") maxlength="5" }}
+                    {{numeric-input id=(concat 'height' index) name="height" value=pkg.height placeholder=(t "placeholder.height") maxlength="5"  pattern="\d{1,5}" }}
```
Comment (steveyken) [inline app/templates/review_item/accept.hbs]: @swatijadhav we should add validation to the API server too.

Comment (swatijadhav) [inline app/templates/review_item/accept.hbs]: yes @steveyken, we will add it. Thanks for reviewing changes :)

**Comment by steveyken** [comment]
I added GCW-706 to cover this work

**Comment by swatijadhav** [comment]
@steveyken Thanks :)

**Pull request merged**